### PR TITLE
🎨 backend pulls inputs before iframe is displayed

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

Allows for inputs to be in place before the iframe is displayed.

Why is this a good idea?
- Some services run code that checks inputs as soon as the iframe is loaded.
- Some users run their scripts as soon as the service is loaded and since currently there is no visual queue about input ports being pulled this helps them to be sure that the inputs are in place.

## Related issue/s

- closes https://github.com/ITISFoundation/osparc-issues/issues/1711

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
:+1:

File: services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_user_services.py [hunk 1]
```diff
@@ -223,6 +223,9 @@ async def progress_create_containers(
 
     scheduler_data.dynamic_sidecar.were_containers_created = True
 
+    # NOTE: user services are already in running state, meaning it is safe to pull inputs
+    await sidecars_client.pull_service_input_ports(dynamic_sidecar_endpoint)
```
Comment (sanderegg) [inline services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_user_services.py:227]: Question: is this connected to the service progress bar?

**Review by sanderegg — APPROVED** [review]
very nice, thanks! that is why maintenance tasks are good.

**Review by pcrespov — APPROVED** [review]

**Review by odeimaiz — APPROVED** [review]
👌

File: services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_user_services.py (hunk 1 above)
Comment (GitHK) [reply services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_user_services.py:227]: yes, the progress bar now fills up

**Pull request merged**